ui/admin/tests/e2e/helpers/boundary-ui.js

```diff
@@ -668,9 +668,10 @@ exports.addMemberToGroup = async (page, userName) => {
 /**
  * Uses the UI to create a new role. Assumes you have selected the desired scope.
  * @param {Page} page Playwright page object
- * @param {string} roleName Name of the new role
  */
-exports.createNewRole = async (page, roleName) => {
+exports.createRole = async (page) => {
+  const roleName = 'Credential ' + nanoid();
```

Comment:
not a blocker totally a nitpick- should this be called `Role`?

Reply:
Yup! Will fix that.